addons/danger/stringtable.xml

```diff
@@ -97,6 +97,22 @@
             <English>Toggles the concealment move by AI for units not equipped to damage tanks and aircraft. Disabling this setting will make groups more responsive</English>
             <German>Schaltet das verdeckte Bewegen von KI für Einheiten, die keine Ausrüstung zu Panzerabwehr oder Flugzeugabwehr besitzen, um. Durch das Ausschalten reagieren die Einheiten schneller.</German>
         </Key>
+        <Key ID="STR_Lambs_Danger_Settings_DisableStaticDeployment">
+            <English>Disable units deploying static weapons</English>
+            <German>TODO</German>
```

Comment:
Do not include todo string tables, translators will add those themselves

Reply:
Fixed. Cheers.